draft-ietf-suit-manifest.cddl

```diff
@@ -85,25 +85,25 @@ suit-cose-hash-algs /= cose-alg-shake256
 
 SUIT_Component_Identifier =  [* bstr]
 
-SUIT_Common_Sequence = [
+SUIT_Shared_Sequence = [
     + ( SUIT_Condition // SUIT_Common_Commands )
```

Comment:
Should "Common_Commands" be "Shared_Commands" as well?

Reply:
Yes, it probably should.